.github/workflows/release.yaml

```diff
@@ -0,0 +1,76 @@
+name: Create release
+on:
+  push:
+    tags:
+      - 'v*'
+  # should be able to not have to create tag manually
+  # workflow_dispatch: {}
+
+env:
+  CONTROLLER: profiles-controller
+  ORG: weaveworks
+
+jobs:
+  # run all tests first?
```

Comment:
if its on main it should be green since it passed PR review :thinking:

Reply:
indeed, one would hope